bench/src/strategies/tc1.rs

```diff
@@ -0,0 +1,88 @@
+use anyhow::Context;
+use solana_rpc_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::transaction::Transaction;
+use solana_sdk::{commitment_config::CommitmentConfig, signature::Keypair};
```

Comment:
Document with comments and with a readme what different strategies are.

Reply:
They are there in the Command Line arguments